CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
-cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
+execute_process(COMMAND "bash" "-c" "module load tools/git dev/gcc-latest dev/cuda11.4 tools/cmake-latest dev/nccl_2.11.4-1_cuda11.4_x86_64 hpcx-gcc-mt; export LD_PRELOAD=/lib64/libcuda.so; export CUDACXX=/hpc/local/oss/cuda11.4/bin/nvcc")
```

Comment:
this change can be reversed, it was just for my personal setup, and I forgot to remove it